src/server/scripts/Spells/spell_item.cpp

```diff
@@ -3590,7 +3591,9 @@ class spell_item_demon_broiled_surprise : public SpellScriptLoader
 
         bool Validate(SpellInfo const* /*spell*/) override
         {
-            if (!sSpellMgr->GetSpellInfo(SPELL_CREATE_DEMON_BROILED_SURPRISE) || !sObjectMgr->GetCreatureTemplate(NPC_ABYSSAL_FLAMEBRINGER) || !sObjectMgr->GetQuestTemplate(QUEST_SUPER_HOT_STEW))
+            return ValidateSpellInfo({ SPELL_CREATE_DEMON_BROILED_SURPRISE });
+
+            if (!sObjectMgr->GetCreatureTemplate(NPC_ABYSSAL_FLAMEBRINGER) || !sObjectMgr->GetQuestTemplate(QUEST_SUPER_HOT_STEW))
                 return false;
             return true;
```

Comment:
same here, check the code after the return

Reply:
I  will remove this, validate is just for spells. This other things doesnt matter